src/helpers/add-pr-stale-label.ts

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2022 Expedia, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { LATE_REVIEW, STALE } from '../constants';
+import { HelperInputs } from '../types/generated';
+import { octokit } from '../octokit';
+import { RestEndpointMethodTypes } from '@octokit/rest';
+
+type PullResponseType = RestEndpointMethodTypes["pulls"]["get"]["response"];
+type ListResponseType = RestEndpointMethodTypes["pulls"]["list"]["response"];
+
+export class AddPrStaleLabel extends HelperInputs {
+  owner = '';
+  repo = '';
+}
+
+/* Adds labels to Pull Requests when they become "Stale" or "Late Review"
+ * @param [Array] prs: array of the pull request ids
+ * @param [string] owner: name of the repo's owner
+ * @param [string] repo: name of the repo
+ */
+export const addPrStaleLabel = async ({ owner, repo }: AddPrStaleLabel) => {
+  // Get all pull requests
+  var pull_requests = await octokit.pulls.list({
+    owner: owner,
+    repo: repo,
+    state: "open"
+  }) as ListResponseType;
+
+  var pull_request_data;
+  var pr;
+  var num_requests = pull_requests.data.length;
+  var label;
+
+  // Loop through all of the issue numbers
+  for (var i = 0; i < num_requests; i++) {
```

Comment:
We use [`Bluebird.map()`](https://www.npmjs.com/package/bluebird) in this repo for multiple async operations